GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -1,18 +1,24 @@
 #!/bin/bash
 
-CONFIGURATION=$1
+SOURCEDIRECTORY=$1
+if [ -z $SOURCEDIRECTORY ]; then
+  echo "Build configuration not specified"
+  exit 1
+fi
+
+CONFIGURATION=$2
 if [ -z $CONFIGURATION ]; then
   echo "Build configuration not specified"
   exit 1
 fi
 
-PACKAGEVERSION=$2
+PACKAGEVERSION=$3
 if [ -z $PACKAGEVERSION ]; then
   echo "Installer package version not specified"
   exit 1
 fi
 
-BUILDOUTPUTDIR=$3
+BUILDOUTPUTDIR=$4
 if [ -z $BUILDOUTPUTDIR ]; then
   echo "Build output directory not specified"
   exit 1
```

Comment:
Not strictly related to this PR, but since you're in here could you update the script to validate that `VFS_OUTPUTDIR` and `VFS_PUBLISHDIR` are set?

I've run into problems where the script starts collecting the wrong files if these are not set.

Reply:
There was a directory exists check in line no. 46 inside `CheckBuildIsAvailable` but that was not working. Fixed the issue. Also added couple string checks for these settings in the beginning as well.